lamindb/curators/__init__.py

```diff
@@ -402,8 +402,18 @@ def validate(self) -> None:
                 self._cat_manager_validate()
             except pandera.errors.SchemaError as err:
                 self._is_validated = False
+                err_msg = str(err)
+
+                # Reduce Pandera validation stdout for many columns
+                if "column" in err_msg and "not in dataframe" in err_msg:
+                    missing_col = err_msg.split("column '")[1].split("'")[0]
+                    display_cols_str = _format_values(
+                        list(self._dataset.columns), n=10, quotes=True, sep="'"
+                    )
+                    err_msg = f"column '{missing_col}' not in dataframe. {len(list(self._dataset.columns))} columns in dataframe including: {display_cols_str}"
```

Comment:
```
err_msg = f"column '{missing_col}' not in dataframe. {len(list(self._dataset.columns))} columns in dataframe: {display_cols_str}"
```

Reply:
what happens if more than 1 columns are not in dataframe?

Reply:
This is OK if you need it urgently, but is a bit patchy, we should ideally have a parser for all pandera errors.